main/src/library/Console.flix

```diff
@@ -91,6 +91,25 @@ namespace Console {
                 esc + "[38;2;1m" + ToString.toString(s) + esc + "[0m"
         }
 
+    ///
+    /// Returns `s` formatted with color `hexCode`.
+    ///
+    pub def hexColor(hexCode: String, s: a): String & Impure with ToString[a] =
+        import static java.lang.Integer.parseInt(String, Int32);
+
+        let hexNumbers = String.replace(from = "#", to = "", hexCode);
+        if(String.length(hexNumbers) == 6) {
+            let red = String.slice(0, 2, hexNumbers);
+            let green = String.slice(2, 4, hexNumbers);
+            let blue = String.slice(4, 6, hexNumbers);
+
+            let color = (parseInt(red, 16), parseInt(green, 16), parseInt(blue, 16));
+
+            Console.rgb(color, ToString.toString(s), isFg = true)
+        } else {
+            ToString.toString(s)
```

Comment:
Should we return a `Result` or `Option` instead of the original string?

Reply:
Returning `Option` is just so cumbersome. Lets just return the original string. Returning option would defeat the purpose here.